terraform/aws/data.tf

```diff
@@ -18,12 +18,9 @@ data "aws_ami" "ubuntu" {
   }
 }
 
-data "aws_vpc" "default" {
-  # Allows dynamic lookup of information about the default VPC.
-  # Specifically it's ID
-  default = true
-}
-
-data "aws_iam_role" "ssm_role" {
-  name = "AmazonSSMRoleForInstancesQuickSetup"
-}
+# ToDo:
+#   - Has Not Been Setup in Test yet
+#   - Uncomment as soon as possible
+# data "aws_iam_role" "ssm_role" {
+#   name = "AmazonSSMRoleForInstancesQuickSetup"
```

Comment:
Maybe make the use of the IAM profile optional since it's a role that does not exist by default and members deploying in AWS might opt not to use it.
Something like
```
data "aws_iam_role" "ssm_role" {
  count = var.iam_role != null ? 1 : 0
  name = var.iam_role
}
```